docs/src/main/sphinx/functions/json.md

```diff
@@ -1138,6 +1138,227 @@ FROM customers
+analyze. Use `json_table` in the `FROM` clause of a `SELECT` statement to create
+a table from JSON data.
+
+```text
+JSON_TABLE(
+    json_input,
+    json_path [ AS path_name ]
+    [ PASSING value AS parameter_name [, ...] ]
+    COLUMNS (
+        column_definition [, ...] )
+    [ PLAN ( json_table_specific_plan )
+      | PLAN DEFAULT ( json_table_default_plan ) ]
+    [ { ERROR | EMPTY } ON ERROR ]
+)
+```
+
+The `COLUMNS` clause supports the following `column_definition` arguments:
+
+```text
+column_name FOR ORDINALITY
+| column_name type
+    [ FORMAT JSON [ ENCODING { UTF8 | UTF16 | UTF32 } ] ]
+    [ PATH json_path ]
+    [ { WITHOUT | WITH { CONDITIONAL | UNCONDITIONAL } } [ ARRAY ] WRAPPER ]
+    [ { KEEP | OMIT } QUOTES [ ON SCALAR STRING ] ]
+    [ { ERROR | NULL | EMPTY { [ARRAY] | OBJECT } | DEFAULT expression } ON EMPTY ]
+    [ { ERROR | NULL | DEFAULT expression } ON ERROR ]
+| NESTED [ PATH ] json_path [ AS path_name ] COLUMNS ( column_definition [, ...] )
+```
+
+`json_input` is a character string or a binary string. It must contain a single
+JSON item.
+
+`json_path` is a string literal containing the path mode specification and the
+path expression. It follows the syntax rules described in
+{ref}`json-path-syntax-and-semantics`.
+
+```text
+'strict ($.price + $.tax)?(@ > 99.9)'
+'lax $[0 to 1].floor()?(@ > 10)'
+```
+
+In the `PASSING` clause, pass values as named parameters that the `json_path`
+expression can reference.
+
+```text
+PASSING orders.totalprice AS O_PRICE,
+        orders.tax % 10 AS O_TAX
+```
+
+Use named parameters to reference the values in the path expression. Prefix
+named parameters with `$`.
+
+```text
+'lax $?(@.price > $O_PRICE || @.tax > $O_TAX)'
+```
+
+You can also pass JSON values in the `PASSING` clause. Use `FORMAT JSON` to
+specify the format and `ENCODING` to specify the encoding:
+
+```text
+PASSING orders.json_desc FORMAT JSON AS o_desc,
+        orders.binary_record FORMAT JSON ENCODING UTF16 AS o_rec
+```
+
+The `json_path` value is case-sensitive. The SQL identifiers are uppercase. Use
+quoted identifiers in the `PASSING` clause:
+
+```text
+'lax $.$KeyName' PASSING nation.name AS KeyName --> ERROR; no passed value found
+'lax $.$KeyName' PASSING nation.name AS "KeyName" --> correct
+```
+
+The `PLAN` clause specifies how to join columns from different paths. Use
+`OUTER` or `INNER` to define how to join parent paths with their child paths.
+Use `CROSS` or `UNION` to join sibling.
+
+`COLUMNS` defines the schema of your table. Each `column_definition` specifies
+how to extract and format your `json_input` value into a relational column.
+
+`PLAN` is an optional clause to control how to process and join nested JSON
+data.
+
+`ON ERROR` specifies how to handle processing errors. `ERROR ON ERROR` throws an
+error. `EMPTY ON ERROR` returns an empty result set.
+
+`column_name` specifies a column name.
+
+`FOR ORDINALITY` adds incremental row numbers to the table, starting at `1`.
```

Comment:
How ... as a separate column .. if so .. whats the column name or how can you access it?